src/scripts/main.js

```diff
@@ -2,7 +2,22 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
+const cellCount = 6;
 
-// write your code here
+people.forEach((person) => {
+  const newRow = table.insertRow();
+
+  const cells = [];
+
+  for (let i = 0; i < cellCount; i++) {
+    cells[i] = newRow.insertCell();
+  }
+
+  cells[0].innerHTML = person.name;
+  cells[1].innerHTML = person.sex;
```

Comment:
`Male`/`Female` should be displayed here:
![image](https://github.com/mate-academy/js_task_generate_table_DOM/assets/86923089/da47c0fc-97af-413a-9eb2-dae5b19ca3b6)